src/test/java/io/strimzi/kafka/bridge/http/HttpCorsTests.java

```diff
@@ -61,6 +62,16 @@
     static KafkaFacade kafkaCluster = new KafkaFacade();
     static MeterRegistry meterRegistry = BackendRegistries.getDefaultNow();
 
+    static JmxCollectorRegistry jmxCollectorRegistry;
+
+    static {
+        try {
+            jmxCollectorRegistry = new JmxCollectorRegistry("");
+        } catch (Exception ex) {
```

Comment:
So if we throw an exception here, it doesn't matter?

Reply:
After my refactoring using a `MetricsReporter` class, we can just set to null the registries instances.